### PR TITLE
Pandas Series operators add, sub, mul, div implementation

### PR DESCRIPTION
**Pull request by akharche**
PR [#135](https://github.com/IntelPython/hpat/pull/135) continue

### CODE REVIEW

**Comment by fschlimb** [comment]
@akharche Please investigate CI failure and fix

**Comment by akharche** [comment]
Implemented ability to do operations with scalars + added tests
Implemented truediv() and floordiv()
Changed to double-quoted strings in docstrings, it is python style

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -229,119 +229,243 @@ def hpat_pandas_series_not_equal_impl(lhs, rhs):
...
+        def hpat_pandas_series_add_impl(lhs, rhs):
+            """
+            Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op5
+            """
+
+            return pandas.Series(lhs._data + rhs._data)
+
+        return hpat_pandas_series_add_impl
+
+    if isinstance(rhs, types.Integer) or isinstance(rhs, types.Float):
```
Comment (shssf) [inline hpat/datatypes/hpat_pandas_series_functions.py]: I'm afraid it covers `A + 5` only. I think we will have other expressions like `5 + A`

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 2]
```diff
@@ -229,119 +229,243 @@ def hpat_pandas_series_not_equal_impl(lhs, rhs):
...
+        def hpat_pandas_series_add_impl(lhs, rhs):
```
Comment (shssf) [inline hpat/datatypes/hpat_pandas_series_functions.py]: Is it better to change `hpat_pandas_series_add_impl` to `hpat_pandas_series_add_number_impl`?

File: hpat/datatypes/hpat_pandas_series_functions.py (hunk 1 above)
Comment (shssf) [reply hpat/datatypes/hpat_pandas_series_functions.py]: Please treat my previous comment as "no sense" due to this is a function `series.add(5)` but not an operator `series + 5.0`
Sorry.
This PR is completed and going to be merged.

File: hpat/tests/test_series.py [hunk 3]
```diff
@@ -482,7 +482,7 @@ def test_series_op4(self):
             pd.testing.assert_series_equal(hpat_func(df.A, 1), test_impl(df.A, 1), check_names=False)
 
     def test_series_op5(self):
-        arithmetic_methods = ('add', 'sub', 'mul', 'div', 'truediv', 'floordiv', 'mod', 'pow')
+        arithmetic_methods = ('add', 'sub', 'mul', 'div', 'truediv', 'floordiv', )
```
Comment (shssf) [inline hpat/tests/test_series.py]: Why `mod` and `pow` deleted from the set of function in this test?

**Pull request merged**